packages/ckeditor5-dev-tests/lib/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -22,8 +22,10 @@ const writer = new commonmark.HtmlRenderer();
 const viewTemplate = fs.readFileSync( path.join( __dirname, 'template.html' ), 'utf-8' );
 
 module.exports = function compileManualTestHtmlFiles( buildDir, manualTestPattern ) {
-	const sourceMDFiles = globSync( path.join( manualTestPattern, '*.md' ) );
-	const sourceDirs = uniq( sourceMDFiles.map( file => path.dirname( file ) ) );
+	const sourceMdFiles = globSync( path.join( manualTestPattern, '*.md' ) );
```

Comment:
So one line below there should be `mdFile` or `MDFile`? Because according to your quote it should be written using the second form, which doesn't look good for me.